src/etc/inc/plugins.inc.d/dhcpd.inc

```diff
@@ -574,6 +576,8 @@ function dhcpd_dhcp4_configure($verbose = false)
 
     killbypid('/var/dhcpd/var/run/dhcpd.pid');
 
+    $unbound_enabled = !empty((string)(new \OPNsense\Unbound\Unbound())->general->enabled);
```

Comment:
probably want to do something about this mid-term through a more straightforward "run" plugin hook. invoking models everywhere will not help the legacy code.

Reply:
(thinking about it it makes sense so we can actually make plugins like bind act in the same way if both unbound and dnsmasq are off)